public-yarn.lock

```diff
@@ -9463,6 +9470,7 @@ __metadata:
     "@babel/plugin-proposal-decorators": ^7.21.0
     "@tldraw/assets": "workspace:*"
     "@tldraw/tldraw": "workspace:*"
+    "@vercel/analytics": ^1.0.1
```

Comment:
Why are we adding `@vercel/analytics` to the open source repo?

Reply:
Good catch; at one point I was experimenting with that as well, I'll yank it.